src/raven.js

```diff
@@ -1391,7 +1391,7 @@ Raven.prototype = {
             data = objectMerge({}, crumb.data);
             for (var j = 0; j < urlProps.length; ++j) {
                 urlProp = urlProps[j];
-                if (data.hasOwnProperty(urlProp)) {
+                if (data.hasOwnProperty(urlProp) && data[urlProp]) {
```

Comment:
My first thought was actually to update `truncate`. I may amend my commit.

By the way, an empty string does not need to be truncated, and testing empty values also handle `null`. So let me know what your preference is.